src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -653,19 +653,28 @@ public String[] getApplicationProtocols() {
      * {@code String} should be exchanged using {@code UTF-8}, the
      * {@code String} should be converted to its {@code byte[]} representation
      * and stored as a byte-oriented {@code String} before calling this method.
+     * For example:
      *
      * <blockquote><pre>
-     *     // MEETEI MAYEK LETTERS HUK UN I (Unicode 0xabcd->0xabcf): 2 bytes
-     *     byte[] bytes = "\u005cuabcd\u005cuabce\u005cuabcf"
-     *             .getBytes(StandardCharsets.UTF_8);
-     *     String HUK_UN_I = new String(bytes, StandardCharsets.ISO_8859_1);
+     *     // Encode 3 Meetei Mayek letters (HUK, UN, I) using Unicode Escapes
+     *     //     0xabcd->0xabcf, 2 Unicode bytes/letter.
+     *     String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
      *
-     *     // 0x00-0xFF:  1 byte
-     *     String rfc7301Grease8A = "\u005cu008A\u005cu008A";
+     *     // Convert into UTF-8 encoded bytes (3 bytes/letter)
+     *     byte[] bytes = HUK_UN_I.getBytes(StandardCharsets.UTF_8);
      *
+     *     // Preserve network byte order by using ISO_8859_1 encoding
+     *     String encodedHukUnI =
+     *         new String(bytes, StandardCharsets.ISO_8859_1);
+     *
+     *     // Also, encode a two byte RFC 8701 GREASE ALPN value
+     *     //     e.g. 0x0A, 0x1A, 0x2A...0xFA
+     *     String rfc8701Grease8A = "\u005cu008A\u005cu008A";
```

Comment:
```
jshell> String rfc8701Grease8A = "\u005cu008A\u005cu008A";
|  Error:
|  illegal escape character
|  String rfc8701Grease8A = "\u005cu008A\u005cu008A";
|                                  ^
|  Error:
|  illegal escape character
|  String rfc8701Grease8A = "\u005cu008A\u005cu008A";
|                                             ^
```

Is a backslash missing betweeen the c and the u?

Reply:
Is this a rendering issue or is there something missing (or superfluous) in these strings?

Reply:
See other comment.  javadoc renders this properly in the resulting HTML.